Documentation/install/guides/kube-router.rst

```diff
@@ -0,0 +1,116 @@
+.. _kops_guide:
+
+****************************
+Using kube-router to run BGP
+****************************
+
+This guide explains how to configure Cilium and kube-router to co-operate to
+use kube-router for BGP peering and route propagation and Cilium for policy
+enforcement and load-balancing.
+
+.. note::
+
+    This feature is regarded as ``tech-preview``. Please provide feedback and
+    file a GitHub issue if you experience any problems.
+
+Deploy kube-router
+##################
+
+Download the kube-router DaemonSet template:
+
+.. code:: bash
+
+    curl -LO https://raw.githubusercontent.com/cloudnativelabs/kube-router/master/daemonset/generic-kuberouter-only-advertise-routes.yaml
```

Comment:
Please use a tag instead of `master`

https://raw.githubusercontent.com/cloudnativelabs/kube-router/v0.2.0-beta.7/daemonset/generic-kuberouter-only-advertise-routes.yaml

Reply:
not addressed